docs/notebooks/MetaOT.ipynb

```diff
@@ -0,0 +1,760 @@
+{
```

Comment:
<p><code>... improves them to find them. </code>sounds a bit strange, could you please rephrase?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Updated the first para to:</p><p> </p><pre class="ql-syntax" spellcheck="false">We interpret the pair of MNIST digits as discrete measures
$\alpha = \sum_{i=1}^{n_a} a_i \delta_{x_i}$ and $\beta = \sum_{j=1}^{n_b} b_j \delta_{y_j}$.
The default Sinkhorn implementation in&nbsp;
[ott.core.sinkhorn.sinkhorn](../_autosummary/ott.core.sinkhorn.sinkhorn.html)
can easily compute their optimal coupling and associated
dual potentials $f$ and $g$ from scratch.
The optimal coupling between the measures can be used
to generate an OT interpolation which shows how the
measures are connected.
</pre>

Reply:
Perfect, thanks!